utils/animate3d.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+import sys
```

Comment:
It looks like the pre-commit hooks were not installed in the repo so some of the code was not automatically linted. Sorry, this is my bad, I need to make it clearer in the documentation. 

Could you please add [pre-commit](https://pre-commit.com/) to your local repo hooks and then run it against all files? It will detect some issues, such as unused imports in this and the next lines.